src/shared/i18n/de/mail.json

```diff
@@ -150,14 +150,14 @@
         "failed": {
             "title": "Identifikation fehlgeschlagen",
             "salutation": "Deine Identifikation ist fehlgeschlagen",
-            "message": "Wir haben festgestellt, dass deine Identifizierung fehlgeschlagen ist. Bitte versuche es über die folgende URL erneut: [url:hier]"
+            "message": "Wir haben festgestellt, dass deine Identifizierung fehlgeschlagen ist. Bitte versuche es über die folgende URL erneut:<br>[url:{urlText}]"
         },
         "reminder": {
             "title": "Vervollständige Verifizierung",
             "salutation": "Vervollständige deine Verifizierung",
             "message": "Uns ist aufgefallen, dass deine Verifizierung noch nicht vollständig abgeschlossen wurde."
         },
-        "next_step": "Um den Online-Identifizierungsprozess zu starten, klicke auf der Payment Seite (Zugang über den \"My DFX\" Button in der App) auf den Refresh Button (roter Pfeil in Kreisoptik) oder [url:hier].",
+        "next_step": "Um den Online-Identifizierungsprozess zu starten, klicke auf der Payment Seite (Zugang über den \"My DFX\" Button in der App) auf den Refresh Button (roter Pfeil in Kreisoptik) oder<br>[url:{urlText}]",
```

Comment:
Change to Services? And replace `Um den Online-Identifizierungsprozess zu starten` with `Um mit deiner Verifizierung fortzufahren` or similar?